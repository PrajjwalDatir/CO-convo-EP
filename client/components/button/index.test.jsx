import React from 'react';
import { mount } from 'enzyme';
import Button from './index';

describe('<Button />', () => {
  it('should match snapshot', () => {
    const wrapper = mount(<Button />);
    expect(wrapper).toMatchSnapshot();
  });
});
