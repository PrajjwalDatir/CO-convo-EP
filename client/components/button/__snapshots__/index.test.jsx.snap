// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button /> should match snapshot 1`] = `
<Button
  className=""
  overrideColors={false}
  type="button"
>
  <button
    className="
        bg-gray-700 hover:bg-gray-800 text-gray-100
        rounded focus:shadow-outline cursor-pointer px-4 py-2 shadow hover:shadow-md 
      "
    type="button"
  />
</Button>
`;
